docs/src/terminology.md

```diff
@@ -88,15 +96,19 @@ See [Proof of History](terminology.md#proof-of-history).
 
 ## epoch
 
-The time, i.e. number of [slots](terminology.md#slot), for which a [leader schedule](terminology.md#leader-schedule) is valid.
+The time, as measured in [slots](terminology.md#slot), for which a [leader schedule](terminology.md#leader-schedule) is valid. On Mainnet, there are 432,000 slots per epoch, which should take 2 days to pass
+
+## epoch boundary 
+
+The first block of a new epoch.
```

Comment:
How about, "The moment between two epochs; after the last slot of one and before the first slot of the next."